olexandr.vakulinskiy/HW2/Task1/Tamagotchi.rb

```diff
@@ -0,0 +1,109 @@
+class Tamagotchi
+
+  attr_accessor :name, :health, :hunger, :thirst, :fatigue
+
+  def initialize(name)
+    @name = name
+    @health = 10
+    @hunger = 2
+    @thirst = 2
+    @fatigue = 0
+  end
+
+  def check_health
+    if @hunger == 0 || @thirst == 0
+      @health += 1
+    elsif @hunger >= 7 || @thirst >= 7
+      @health -= 1
+    end
+    if @fatigue == 10
+      @health = 0
+    elsif  @fatigue >=7
+      @health -= 2
+    end
+
+  end
+  def feed
+    check_health
+    @hunger -= 1
+  end
+  def drink
+    check_health
+    @thirst -= 1
+  end
+  def play
+    check_health
+    @fatigue += 2
+    @hunger += 1
+    @thirst += 1
+
+  end
+  def sleep
+    check_health
+    @fatigue -= 1
+    @hunger += 2
+    @thirst += 2
+  end
+
+  def is_dead?(action_with_pet)
+    if @health <= 0
+      true
+    else
+      puts action_with_pet
+      false
+    end
+  end
+  def get_characteristics
+    puts "PET CHARACTERISTICS
+Name: #{name}.
```

Comment:
please, use paddings for multi-line strings or [use heredoc](https://blog.saeloun.com/2020/04/08/heredoc-in-ruby-and-rails.html)
```suggestion
    puts "PET CHARACTERISTICS
             Name: #{name}.
             ...
```